### ISSUE TITLE
[BITV] 9.1.4.11/3.4 - The "Favorites" yellow star does not meet the contrast requirements for graphical informative elements. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36972]
- [ ] Fix
- [ ] Backport

![5e2cbbfe0fd4258eab94ad65371fb1fa](https://user-images.githubusercontent.com/1315170/222242397-1682fd7f-61e4-46af-a82a-5f7734bb2586.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3dbfb7615a-v3-n4

</details>

- [ ] 9.1.4.11/10.4 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3dbfb7615a-v10-n4
